manifests/o/OpenSC/OpenSC/0.23.0/OpenSC.OpenSC.locale.en-US.yaml

```
@@ -0,0 +1,24 @@
+# Created using wingetcreate 1.5.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.5.0.schema.json
+
+PackageIdentifier: OpenSC.OpenSC
+PackageVersion: 0.23.0
+PackageLocale: en-US
+Publisher: OpenSC
+PublisherUrl: https://github.com/OpenSC
+PublisherSupportUrl: https://github.com/OpenSC/OpenSC/issues
+PackageName: OpenSC
+PackageUrl: https://github.com/OpenSC/OpenSC/releases
+License: LGPL-2.1 License
+LicenseUrl: https://github.com/OpenSC/OpenSC/blob/master/COPYING
+ShortDescription: Open Source smartcard driver for Windows
+Description: OpenSC provides a set of libraries and utilities to work with smart cards. Its main focus is on cards that support cryptographic operations, and facilitate their use in security applications such as authentication, mail encryption and digital signatures. OpenSC implements the standard APIs to smart cards, e.g. PKCS#11 API, Windows’ Smart Card Minidriver and macOS CryptoTokenKit.
+Moniker: OpenSC
+Tags:
+- OpenSC

```

```suggestion
- c
- security
- smartcard
- pkcs11
- tokend
- minidriver
- opensc
```

The general convention is that tags should be written in lowercase. Suggested some more tags from https://github.com/OpenSC/OpenSC